src/index.html

```diff
@@ -456,7 +456,7 @@ <h1 class="hero__heading">Hi, I'm Michal</h1>
       <section id="skills" class="skills-section">
         <div data-technology-box  class="technology">
           <ul class="technology__list">
-            <li><i class="devicon-html5-plain colored"></i></li>
+            <li><em class="devicon-html5-plain colored"></em></li>
```

Comment:
@Malposti looks good, can You apply for all <i> occuracnes

Reply:
@Postimal  Done

Reply:
@Malposti I can notice mistake in list tags, please check it out. Seems that <lem> is not correct

Reply:
Fixed